omop/v4/schema/observation_period.csv

```diff
@@ -0,0 +1,5 @@
+model,version,table,field,type
```

Comment:
Yes I thought about it. I will add them since it makes the files themselves more descriptive rather than relying on documentation or code to declare the other supported fields.